src/migrate.ts

```diff
@@ -90,17 +91,16 @@ export default async (
           await copy(migrations, `${TEMP_MIGRATION_DIR}/migrations`)
           await hasuraConsole('migrate apply')
         }
-        // * Metadata is used in config v2. HBP uses config v1 so far
-        // if (metadata && (await pathExists(metadata))) {
-        //   // * Apply metadata
-        //   console.log(`Applying metadata '${metadata}'...`)
-        //   await copy(metadata, `${TEMP_MIGRATION_DIR}/metadata`)
-        //   await hasuraConsole('metadata apply')
-        // } else if (migrations && (await pathExists(migrations))) {
-        console.log('Reloading metadata...')
-        await hasuraConsole('metadata reload')
-        // }
+
+        if (metadata && (await pathExists(metadata))) {
+          // * Apply metadata
+          console.log(`Applying metadata '${metadata}'...`)
+          await copy(metadata, `${TEMP_MIGRATION_DIR}/metadata`)
+          await hasuraConsole('metadata apply')
```

Comment:
What happens with the Hasura metadata if the following happens:

1. Start clean new project
2. First run of HBP
3. HBP run migrations + metadata

All good. Then the following:

4. Add a table and track that table (update the metadata)
5. Turn off HBP
6. Turn on HBP
7. HBP tries to run migrations (not needed, already applied)
8. HBP tries to apply metadata? If yes, the metadata from step 4. is lost?

Reply:
HBP is not responsible for applying metadata. Applying metadata to hasura happens outside of HBP.

Base metadata for HBP will be provided in this repo.